api/v1alpha1/vaultdynamicsecret_types.go

```diff
@@ -64,6 +69,8 @@ type VaultDynamicSecretStatus struct {
 	LastGeneration int64 `json:"lastGeneration"`
 	// SecretLease for the Vault secret.
 	SecretLease VaultSecretLease `json:"secretLease"`
+	// StaticCredsMetaData contains the static creds response meta-data
+	StaticCredsMetaData VaultStaticCredsMetaData `json:"staticMeta,omitempty"`
```

Comment:
Nit: maybe makes sense to have the json be closer to the name to be descriptive? `staticCredsMeta`?

```suggestion
	StaticCredsMetaData VaultStaticCredsMetaData `json:"staticCredsMeta,omitempty"`
```

Reply:
Fixed with 5d7596d16fb034a28f70aca377d804331a0bd301